.github/workflows/build.yaml

```diff
@@ -0,0 +1,61 @@
+name: build
+
+on: 
+  push: 
+    branches:
+      - "*"
+  pull_request: 
+    branches: 
+      - "*"
+  workflow_dispatch: 
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+
+    - name: Install dependencies
+      run: pip install wheel setuptools
+
+    - name: Build wheel
+      run: python setup.py bdist_wheel
+
+    - name: Upload Python wheel
+      uses: actions/upload-artifact@v2
+      with:
+        name: Python wheel
+        path: ${{github.workspace}}/dist/sapai-*.whl
+        if-no-files-found: error
+
+  test:
+    needs: build
+    runs-on: ${{matrix.os}}
+    strategy:
+      max-parallel: 10
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, 3]
```

Comment:
Do all these python versions need to be actively supported? My concern is that this increases the cost of running CI. Is there a strong reason against only supporting the latest two version, say 3.8 and 3.9 for now? 

Thanks for your effort.

Reply:
That's a good question. In general, it is a good idea to run tests for different python versions, but for this project it is likely not necessary. If we build against the oldest version of interest (Python 3.6), it should be forward compatible. Meaning that it should not be necesary to run tests for the other python versions.

So I propose to only build a binary using 3.6 and run tests using the same version. But we should likely run tests for all three operating systems. Hence, in total there will be four jobs (1 build + 3 tests). I can fix that. Sounds good?

However, when we are making tags (in the future) it is probably a good idea to run all these tests (for all relevant python versions) before release, but it does not need to be done for every commit. I have done this in a separate project, and could add this at a later stage, if necessary (see [here](https://github.com/EIDOSLAB/torchstain/tree/main/.github/workflows) for example of how I did it).

Reply:
I have fixed this in the latest commit [ef4990d](https://github.com/manny405/sapai/pull/78/commits/ef4990dd520d9a59a28ed0882587180c19d177fd)

Reply:
This is great, thanks for the info.